src/Symfony/Component/Templating/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.4.0
+-----
+
+* deprecated the component: use Twig instead
```

Comment:
```suggestion
6.4
---

 * Deprecate the component, use Twig instead
```

Our code-style has changed since the last time we did change something in that component. 🙈